plugins/modules/aci_bd_subnet.py

```diff
@@ -367,7 +374,7 @@ def main():
         route_profile=dict(type="str"),
         route_profile_l3_out=dict(type="str"),
         scope=dict(type="list", elements="str", choices=["private", "public", "shared"]),
-        subnet_control=dict(type="str", choices=list(SUBNET_CONTROL_MAPPING_BD_SUBNET.keys())),
+        subnet_control=dict(type="list", elements="str", choices=list(SUBNET_CONTROL_MAPPING_BD_SUBNET.keys())),
```

Comment:
```suggestion
        subnet_control=dict(type="list", elements="str", choices=list(SUBNET_CONTROL_MAPPING_BD_SUBNET)),
```

Reply:
not sure why you want me to change this when it is existing code, but changed it since it makes no real difference